Cargo.toml

```diff
@@ -38,8 +38,7 @@ rayon = { version = "1.2", optional = true }
 
 [dependencies.hashbrown]
 version = "0.11"
-default-features = false
-features = ["raw"]
+features = ["ahash", "raw"]
```

Comment:
Ahash pulls in a non-trivial amount of dependencies. On linux it pulls in libc, version_check, cfg-if, once_cell and getrandom.

Reply:
We can only do that if we make the features mutually exclusive, as a hard build error. You could have one part of a large dependency tree that wants `indexmap` one way, and another part wanting the other way, but Cargo unifies dependencies into one build so we can't satisfy both.

But that's only about default `S` and the few methods that assume that. Most of the API is perfectly fine with bring-your-own-hasher `S`, so each part of that dependency tree can fill their own needs.

Reply:
Yeah, I had used that approach of pseudo-mutually-exclusive features for another project via the `compile_error!()` macro, but it's admittedly tedious and feels hacky. 

I do hear what you're saying about multiple downstream crates using `indexmap` differently, and unfortunately there's no way around that.